packages/aws-cdk/bin/cdk.ts

```diff
@@ -238,7 +238,7 @@ async function initCommandLine() {
 
       case 'init':
         const language = configuration.settings.get(['language']);
-        const generateOnly = configuration.settings.get(['generate-only']);
+        const generateOnly = configuration.settings.get(['generateOnly']);
```

Comment:
`args.generateOnly` would also be fine here

Reply:
Should I remove `generateOnly` from the settings completely then?
Doesn't seem to be used anywhere else.